components/Header.tsx

```diff
@@ -41,7 +42,8 @@ export const Header = ({
   const [scheduleVisible, setScheduleVisible] = useState(true);
   const [drawerVisible, setDrawerVisible] = useState(false);
   const router = useRouter();
-
+  const LINK_CLASSES =
```

Comment:
Constants should not be located within the component function. They should be placed outside of it.

Reply:
not needed anymore as only calling it once